js/components/multi-select.js

```diff
@@ -14,6 +14,7 @@ define(
 
       self.multiple = params.multiple;
       self.options = params.options;
+	  self.disable = (params.disable || false);
```

Comment:
should this be `self.disabled` to match `attr: disabled,` added to the .html above?